frontend/src/app/shared/components/op-context-menu/op-context-menu.types.ts

```diff
@@ -20,6 +20,7 @@ export interface OpContextMenuLocalsMap {
   showAnchorRight?:boolean;
   contextMenuId?:string;
   label?:string;
+  visibleLabel?:boolean;
```

Comment:
I was a super confused by the `visibleLabel` option. First, I expected `visibleLabel` to be of type string and not a boolean. A boolean would be named `labelVisible`. Second, I expected these headlines to be menu items with a special flag rather then an option for the whole menu. Imagine you need a second headline in the menu, that will not work with your current approach.

So my suggestion would be to create an attribute for the items (something like `isHeader?:boolean`) and add the header as one of the items for the context menu.